routes/activities/read.js

```diff
@@ -12,11 +13,23 @@ const handleRead = async (req, res, reader) => {
         res.status(404).send(`document with id ${body.object.id} not found`)
         break
       }
-      const activityObjStack = createActivityObject(body, resultDoc, reader)
-      Activity.createActivity(activityObjStack)
+
+      // const activityObjStack = createActivityObject(body, resultDoc, reader)
+      const object = {
+        selector: body['oa:hasSelector']
+      }
+
+      if (body.json) {
+        object.json = body.json
+      }
```

Comment:
This works but you could make it shorter by doing 
object = {
selector: body['oa:hasSelector'],
json: body.json
}

if there is no json, it will just be undefined. it's ok.